bindata/network/multus/multus.yaml

```diff
@@ -1,3 +1,82 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fips-binary-copy-script
+  namespace: openshift-multus
+  annotations:
+    kubernetes.io/description: |
+      This is a script used to copy binaries in a FIPS compliant fashion
```

Comment:
"copy binaries in a FIPS compliant fashion" makes it sound like you're using some cryptographically-secure version of `/usr/bin/cp`. This script really has nothing do to with FIPS per se, it's just that for FIPS reasons, we need separate RHEL7 and RHEL8 copies of CNI plugins, and this deals with that. I'd rename it all to be about installing CNI plugins rather than FIPS. And you can get rid of `DESTINATION_DIRECTORY`; it's always `/host/opt/cni/bin`.

Meanwhile, see also https://issues.redhat.com/browse/SDN-695. This script solves some of the problems there, though it still requires each daemonset to mount the right underlying files/dirs onto `/host/opt/cni/bin`, `/host/etc/os-release`, etc

Reply:
+1, yeah I'll change the language here to talk about cni binary copy based on host OS, makes more sense.